### PR TITLE
Integrate functions for tags from hooks.go files and add unit test for it

### PR DESCRIPTION
**Pull request by kyriechen96**
Issue #, if available:

Description of changes:
Integrate functions for tags from hooks.go files and add unit test for it.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/util/tags.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	}
+	for _, desiredElement := range desired {
+		if !ackutil.InStrings(*desiredElement.Key, visitedIndexes) {
+			addedOrUpdated = append(addedOrUpdated, desiredElement)
+		}
+	}
+	return addedOrUpdated, removed
+}
+
+func SdkTagsFromResourceTags(
```
Comment (RedbackThomson) [inline pkg/util/tags.go:42]: Please update to `SDK` to match the other
```suggestion
func SDKTagsFromResourceTags(
```

**Review by RedbackThomson — CHANGES_REQUESTED** [review]
Just the one comment

File: pkg/util/tags.go (hunk 1 above)
Comment (kyriechen96) [reply pkg/util/tags.go]: Updated!

**Comment by RedbackThomson** [comment]
/hold
test hold

**Comment by kyriechen96** [comment]
There were changes to the code-generator that added support for the tags comparison.

**Pull request closed without merging**